templates/blog/post_list.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{% block content %}
+  {% for post in post_list %}
+    <div class="post">
+      <p><b>Title</b> : <a href="{% url 'blog:post-detail' post.id %}">{{ post.title }}</a></p>
```

Comment:
For better readability, consider using a more descriptive label instead of 'Title'. For example, you could use 'Post Title' or 'Blog Post Title'.